src/Ocis.php

```diff
@@ -306,6 +306,28 @@ private function getServiceUrlFromWebfinger(string $webfingerUrl): string
         }
         throw new InvalidResponseException('invalid webfinger response');
     }
+
+    /**
+     * returns current oCIS version in semantic versioning format ( eg "5.0.5" )
+     *
+     * @return string
+     * @throws InvalidResponseException
+     */
+    public function getOcisVersion(): string
+    {
+        $fullUrl = $this->getServiceUrl() . '/ocs/v1.php/cloud/capabilities';
+        $client = new Client($this->createGuzzleConfig($this->connectionConfig, $this->accessToken));
```

Comment:
maybe we could use https://github.com/owncloud/ocis-php-sdk/pull/229/files#diff-82277ffad27771b61ef747676ab6b941eff7ed78d53b2633a6e5c29fe202309bR102